src/libraries/Common/src/System/Security/Cryptography/ECDiffieHellmanOpenSsl.cs

```diff
@@ -9,7 +9,7 @@ namespace System.Security.Cryptography
 {
     public sealed partial class ECDiffieHellmanOpenSsl : ECDiffieHellman
     {
-        private ECOpenSsl? _key;
+        private SafeEvpPKeyHandle _key;
```

Comment:
This should also be `Lazy<SafeEvpPKeyHandle>` otherwise imports will need to generate key before importing